README.md

```diff
@@ -16,47 +16,100 @@ Based on `phoenixrtos/build` - additional development tools added.
 
-# building will take about approximately 15 hours for arm/v7 and 1.5 hours for amd64
-# - don't build armv7 images for build, devel (only for gh-runner)
-# - after building test if it builds all TARGETs from phoenix-rtos-project
 # use --load flag to load image into docker but it works only for one platform
-docker buildx build --load --platform linux/arm/v7 -t phoenixrtos/$IMAGE_NAME:armv7 -f $IMAGE_NAME/Dockerfile $IMAGE_NAME
-docker buildx build --load --platform linux/amd64 -t phoenixrtos/$IMAGE_NAME:amd64 -f $IMAGE_NAME/Dockerfile $IMAGE_NAME
-
-# push images to the docker hub repo
-docker push phoenixrtos/$IMAGE_NAME:armv7
+docker buildx build --load --progress=plain --platform linux/amd64 -t phoenixrtos/$IMAGE_NAME:amd64 -f $IMAGE_NAME/Dockerfile $IMAGE_NAME
+docker buildx build --load --progress=plain --platform linux/arm64 -t phoenixrtos/$IMAGE_NAME:arm64 -f $IMAGE_NAME/Dockerfile $IMAGE_NAME
+ 
+# push images to the DockerHub repo #
 docker push phoenixrtos/$IMAGE_NAME:amd64
+docker push phoenixrtos/$IMAGE_NAME:arm64
 
 # create manifest composed of images of different architectures
 # WARN: only remote images are taken into account, -amend doesn't really work
 docker manifest rm phoenixrtos/$IMAGE_NAME
-docker manifest create phoenixrtos/$IMAGE_NAME:latest phoenixrtos/$IMAGE_NAME:armv7 phoenixrtos/$IMAGE_NAME:amd64
+docker manifest create phoenixrtos/$IMAGE_NAME:latest phoenixrtos/$IMAGE_NAME:amd64 phoenixrtos/$IMAGE_NAME:arm64
+
 # push the image
 docker manifest push phoenixrtos/$IMAGE_NAME:latest
 
 # create the same manifest with the current date
-docker manifest create phoenixrtos/$IMAGE_NAME:$DATE phoenixrtos/$IMAGE_NAME:armv7 phoenixrtos/$IMAGE_NAME:amd64
+docker manifest create phoenixrtos/$IMAGE_NAME:$DATE phoenixrtos/$IMAGE_NAME:amd64 phoenixrtos/$IMAGE_NAME:arm64
 docker manifest push phoenixrtos/$IMAGE_NAME:$DATE
 
-# create new tag in the repo
-git tag -a $DATE
+### push images to the Github Registry ###
+docker image tag phoenixrtos/$IMAGE_NAME:amd64 ghcr.io/phoenix-rtos/$IMAGE_NAME:amd64
+docker push ghcr.io/phoenix-rtos/$IMAGE_NAME:amd64
+docker image tag phoenixrtos/$IMAGE_NAME:arm64 ghcr.io/phoenix-rtos/$IMAGE_NAME:arm64
+docker push ghcr.io/phoenix-rtos/$IMAGE_NAME:arm64
 
+# WARN: only remote images are taken into account, -amend doesn't really work
+docker manifest rm ghcr.io/phoenix-rtos/$IMAGE_NAME
+docker manifest create ghcr.io/phoenix-rtos/$IMAGE_NAME:latest ghcr.io/phoenix-rtos/$IMAGE_NAME:arm64 ghcr.io/phoenix-rtos/$IMAGE_NAME:amd64
+# push the image
+docker manifest push ghcr.io/phoenix-rtos/$IMAGE_NAME:latest
+
+# for now don't push version-tagged manifests to ghcr.io
+
+# create new tag in the repo
+git tag -a $DATE -m "release: $DATE"
 ```
 
-Repeat the above steps for:
+### amd64 images
+
+build, devel
 
 ```bash
-IMAGE_NAME=devel
-IMAGE_NAME=gh-runner
+IMAGE_NAME=build
+DATE=$(date +%Y%m%d) # or v3.2.1 ( DATE=$PH_REF )
+PH_REF="HEAD" # set explicit phoenix-rtos-project commit id/tag
```

Comment:
obsolete, use `PHOENIX_PROJECT_REF`. Should also be included in `gh-runner` description